try.go

```diff
@@ -129,7 +129,14 @@ type wrapError struct {
 }
 
 func (e wrapError) Error() string {
-	return e.frame.File + ":" + strconv.Itoa(e.frame.Line) + ": " + e.error.Error()
+	file := e.frame.File
+	for i := len(file) - 1; i >= 0; i-- {
```

Comment:
Add a comment here so people don't have to think too hard about what this is doing. (And note there that we don't want  `strings` dependency, which would make this code simpler.)

Reply:
Done.